src/app/auth-form/auth-form.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormControl, FormGroup, NgForm } from '@angular/forms';
```

Comment:
Please use [FormBuilder](https://angular.io/guide/reactive-forms#introduction-to-formbuilder)